hat/assets/js/apps/Iaso/domains/orgUnits/components/OrgUnitInfosComponent.js

```diff
@@ -211,77 +232,107 @@ const Actions = (
-            />
-            <InputComponent
-                keyValue="created_at"
-                value={moment.unix(orgUnit.created_at).format('LTS')}
-                disabled
-            />
-            <InputComponent
-                keyValue="updated_at"
-                value={moment.unix(orgUnit.updated_at).format('LTS')}
-                disabled
-            />
-           
-
-            {!orgUnit.has_geo_json && !latitudeLongitude && (
-                <Grid
-                    container
-                    spacing={1}
-                    className={classes.geometryExistence}
-                >
-                    <Grid item>
-                        <GpsOffIcon color="primary" />
-                    </Grid>
-                    <Grid item>
-                        <Typography>
-                            <FormattedMessage
-                                {...MESSAGES.hasNoGeometryAndGps}
-                            />
-                        </Typography>
-                    </Grid>
-                </Grid>
-            )}
+            {!orgUnit && <LoadingSpinner absolute />}
 
-            {orgUnit.has_geo_json && (
-                <Grid
-                    container
-                    spacing={1}
-                    className={classes.geometryExistence}
-                >
-                    <Grid item>
-                        <GpsFixedIcon color="primary" />
-                    </Grid>
-                    <Grid item>
-                        <Typography>
-                            <FormattedMessage {...MESSAGES.hasGeometry} />
-                        </Typography>
-                    </Grid>
-                </Grid>
-            )}
-            {latitudeLongitude && (
-                <InputComponent
-                    keyValue="latitudeLongitude"
-                    value={latitudeLongitude}
-                    type="text"
-                    disabled
-                    label={MESSAGES.latitudeLongitude}
-                />
-            )}
+            <WidgetPaper showHeader={false} title="">
+                <Table size="medium">
+                    <TableBody>
+                        <TableRow>
+                            <TableCell className={classes.leftCell}>
+                                {formatMessage(MESSAGES.source)}
+                            </TableCell>
+                            <TableCell>{orgUnit.source ?? '-'}</TableCell>
+                        </TableRow>
+
+                        {orgUnit.creator.value && (
+                            <TableRow>
+                                <TableCell className={classes.leftCell}>
+                                    {formatMessage(MESSAGES.creator)}
+                                </TableCell>
+                                <TableCell>{orgUnit.creator.value}</TableCell>
+                            </TableRow>
+                        )}
+
+                        <TableRow>
```

Comment:
We already have a component for that `TableRow`, also you can loop trough a constant to avoid code repetition